pkg/controllers/deprovisioning/singlemachineconsolidation.go

```diff
@@ -62,6 +65,7 @@ func (c *SingleMachineConsolidation) ComputeCommand(ctx context.Context, candida
 			continue
 		}
 
+		v := NewValidation(consolidationTTL(cmd.candidates), s.clock, s.cluster, s.kubeClient, s.provisioner, s.cloudProvider, s.recorder)
```

Comment:
We have to create a new validation for each new set of candidates since that set of candidates may have a different max consolidationTTL

Reply:
Will be nice to have this identified before this even gets to the deprovisioning controller so that we don't have to do this.

Reply:
How do you mean? This value is dependent on the subset of candidates that we are trying to consolidate so it could change for each individual bespoke set.